make/tsb.sh

```diff
@@ -78,7 +78,11 @@ function deploy_mp_fqdn() {
   echo "Processing cluster:" 
   echo "${cluster}" | jq '.'
 
-  run "pushd tsb/fqdn/${dns_provider} > /dev/null"
+  if [[ "${dns_provider}" == aws* ]]; then
```

Comment:
Hmm if you provide `dns_provider` in variables file this shouldn't happen.
`tetrate.dns_provider`  I almost wonder if we should make dns_provider required, or our `variables.sh` should narrow this down more effectively not the `tsb.sh`